README.md

```diff
@@ -5,10 +5,18 @@ This extension adds language support for Ansible to
 and [OpenVSX](https://open-vsx.org/extension/redhat/ansible) compatible editors
 by leveraging [ansible-language-server](https://github.com/ansible/ansible-language-server).
 
-Keep it mind that due to [ALS#117](https://github.com/ansible/ansible-language-server/issues/117)
-this extension works only when a workspace is present. If you open a standalone
-file outside a vscode workspace, you will not be able to use most of the
-features of this extension.
+## Activating Red Hat Ansible extension
+
+It is recommended to open folder containing Ansible files with VSCode workspace.
+
+![Linter support](https://github.com/ansible/ansible-language-server/raw/main/images/ansible-extension-activation.gif)
```

Comment:
As a half-measure, I've uploaded it to GH CDN for you:
```suggestion
![Linter support](https://user-images.githubusercontent.com/578543/143657900-66789b17-ccda-4588-8945-1c936a849532.gif)
```
If you go for this and drop adding the blob from the PR, I think we could settle on the lower quality GIF for some time. The important thing is that it does not affect the Git repo.